AppVeyor is flaking out again, error is unrelated to these changes.

```
npm ERR! path C:\projects\uportal-home\node_modules\requirejs\bin\r.js.1308609769
npm ERR! code EPERM
npm ERR! errno -4048
npm ERR! syscall rename
npm ERR! Error: EPERM: operation not permitted, rename 'C:\projects\uportal-home\node_modules\requirejs\bin\r.js.1308609769' -> 'C:\projects\uportal-home\node_modules\requirejs\bin\r.js'
```

https://ci.appveyor.com/project/ChristianMurphy/uportal-home/build/1.0.136#L28

--- a/package.json
+++ b/package.json
@@ -81,6 +81,9 @@
     "stylelint": "^8.0.0",
     "stylelint-config-standard": "^18.0.0"
   },
+  "engines": {
+    "npm": ">=5.6.0"
+  },
   "stylelint": {
     "extends": "stylelint-config-standard",
     "rules": {

